lib/src/unix_socket.rs

```diff
@@ -0,0 +1,119 @@
+//! hyperlocal provides [hyper](http://github.com/hyperium/hyper) client and server bindings
```

Comment:
Is this exactly the code from `hyper_local`?

Reply:
It's [hyperlocal v0.3.0](https://github.com/softprops/hyperlocal/blob/0.3.0/src/lib.rs) with unused code removed thus removed `rustc-serialize` dependency.